schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+id INT,
+name TEXT,
```

Comment:
[OPTIONAL]
- [ ] I suggest you change this from `TEXT` to a `VACHAR()` where you can enter a maximum length of characters to be entered into the brackets.